pkg/controller/server_status_request_controller.go

```diff
@@ -85,14 +90,26 @@ func (r *ServerStatusRequestReconciler) Reconcile(req ctrl.Request) (ctrl.Result
 	case "", velerov1api.ServerStatusRequestPhaseNew:
 		log.Info("Processing new ServerStatusRequest")
 
-		if err := r.ServerStatus.PatchStatusProcessed(r.Client, statusRequest, r.Ctx); err != nil {
-			log.WithError(err).Error("Unable to update the request")
+		// Initialize the patch helper.
+		patchHelper, err := patch.NewHelper(statusRequest, r.Client)
+		if err != nil {
+			log.WithError(err).Error("Error getting a patch helper to update this resource")
+			return ctrl.Result{}, err
```

Comment:
It's rare that you wouldn't want to re-queue on failure. One case is when the object is not found: most likely it will continue not to be found. In this case, return a nil instead of an error.

Anytime an error is return, the request will be re-queued.